### PR TITLE
Fix spacing in Cleanup dialog #10081

### LINKED ISSUE #10081: Fix spacing in Cleanup dialog
**Is your suggestion for improvement related to a problem? Please describe.**
The cleanup dialog looks ugly!

**Describe the solution you'd like**
Add some spacing around (top, bottom, left, right). See other dialogs like manage study definition for an example. 

**Additional context**

grafik

CleanupDialog is the code related starting point 

Refs #https://github.com/JabRef/jabref-issue-melting-pot/issues/237

### PR DESCRIPTION
**Pull request by pcabaniss**
Fixes #10081 , Updated fxml file in Scenebuilder. Fixed spacing and indentation.

Screenshot 2023-07-15 at 2 55 29 AM

### Mandatory checks
- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [x] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: .vscode/launch.json
Comment (calixtus) [inline .vscode/launch.json:15]: This is probably an unwanted artifact

Comment (pcabaniss) [reply .vscode/launch.json:15]: Sorry, this is the first contribution I've ever done. Is this bad practice?

Comment (calixtus) [reply .vscode/launch.json:15]: No problem, we are here to help! 😍

**Comment by calixtus** [comment]
Great job for your first contribution!
Just made some small whitespace changes.
Just as a suggestion: If you want to create changes, first create a new branch instead of directly commiting on your main branch. This way you can always keep multiple branches of you are working on multiple projects and you always have your main branch in sync with the official repo.

**Comment by pcabaniss** [comment]
Thanks! I'll remember that for next time. If there's any other criticism, please let me know.😁

**Review by calixtus — APPROVED** [review]
Changes look good to me. LGTM :+1:. The changes need a second review and thus this PR awaits a second review.

**Comment by calixtus** [comment]
Failing unit test is not related to your changes. Fixed by #10086

**Review by Siedlerchr — DISMISSED** [review]
Thank you very much for your contribution! A small change with huge impact!
Can you please also add an entry to the Changelog.md ?

**Comment by pcabaniss** [comment]
I changed it and committed, do I need to make another PR? Or does it automatically merge with this one?

**Comment by Siedlerchr** [comment]
Just commit and push your commits, the PR will be updated automatically. 
When merging, we "squash" all commits into one.

**Comment by pcabaniss** [comment]
Done. Thank you both for your help and patience.

**Comment by calixtus** [comment]
After committing the changes, you still have to push the commits to the remote (upstream) repo. 'git push'

**Comment by Siedlerchr** [comment]
PS: We recommend Intellij for setting up your workspace https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/

**Review by Siedlerchr — APPROVED** [review]

**Comment by Siedlerchr** [comment]
Thanks again for your contribution! Also make sure to create a new branch the next time, makes it easier

**Pull request merged**
